targets/targets.json

```diff
@@ -3154,6 +3178,10 @@
             "usb_speed": {
                 "help": "USE_USB_OTG_FS or USE_USB_OTG_HS or USE_USB_HS_IN_FS",
                 "value": "USE_USB_OTG_HS"
+            },
+            "i2c_timing_value_algo": {
+                "help": "If value was set to true I2C timing algorithm is enabled. Enabling may leads to performance issue. Keeping this false and changing system clock will trigger assert.)",
+                "value": true
```

Comment:
I don't think this is working ?
As PORTENTA_H7 is inheriting from MCU_STM32H7, "i2c_timing_value_algo" config already exist,
so I think you need to overwrite the value to true in the "overrides" part below ?

Reply:
Moved to overrides
```
@@ -3178,10 +3178,6 @@
             "usb_speed": {
                 "help": "USE_USB_OTG_FS or USE_USB_OTG_HS or USE_USB_HS_IN_FS",
                 "value": "USE_USB_OTG_HS"
-            },
-            "i2c_timing_value_algo": {
-                "help": "If value was set to true I2C timing algorithm is enabled. Enabling may leads to performance issue. Keeping this false and changing system clock will trigger assert.)",
-                "value": true
             }
         },
         "macros_add": [
@@ -3202,7 +3198,8 @@
             "clock_source": "USE_PLL_HSE_EXTC",
             "lse_available": 1,
             "lpticker_delay_ticks": 0,
-            "network-default-interface-type": "ETHERNET"
+            "network-default-interface-type": "ETHERNET",
+            "i2c_timing_value_algo": true
         },
         "device_name": "STM32H747XIHx"
     },

```